README.md

```diff
@@ -45,6 +45,17 @@ To download azion CLI through Homebrew, run:
 brew install azion
 ``````
 
+##### Test The Azion CLI with docker 
+get link release https://github.com/aziontech/azion/releases
```

Comment:
You can find the URL for each package in our Releases page https://github.com/aziontech/azion/releases